zip-proportional-output-fee-mechanism-pofm.rst

```diff
@@ -229,67 +219,53 @@ Wallet developers and operators should monitor the Zcash network for rapid
 growth in transaction rates.
 
 
-Transaction relaying
---------------------
-
-zcashd, and potentially other node implementations, implement fee-based
-restrictions on relaying of mempool transactions. Nodes that normally relay
-transactions are expected to do so for transactions that pay at least the
-conventional fee, unless there are other reasons not to do so for robustness
-or denial-of-service mitigation.
-
-
-Mempool size limiting
----------------------
-
-zcashd and Zebra limit the size of the mempool as described in [#zip-0401]_. This
-specifies a *low\_fee\_penalty* that is added to the "eviction weight" if the
-transaction pays a fee less than the `min_fee` specified by this ZIP.
-
-
 Deployment
 ==========
 
-The height for coordinating deployment is block 1,800,000 for Mainnet, or immediately on 
-implementing this ZIP for Testnet.
+Wallets can deploy these changes immediately. Nodes can deploy restrictions
+to their policies for relaying, mempool acceptance and eviction, and/or
+mining once a sufficient proportion of wallets in the ecosystem are observed
+to be paying at least the updated conventional transaction fee.
```

Comment:
Reminder: changing the low-fee threshold for ZIP 401 mempool limiting could be deployed before relay or mining policy changes.